examples/ecs/system_closures.rs

```diff
@@ -0,0 +1,29 @@
+use bevy::prelude::*;
+
+fn main() {
+    let arg = "hello";
+    App::build()
+        .add_system(Box::new(move |mut cmd: Commands| {
+            println!("this system uses the moved arg: {:?}", arg);
```

Comment:
```suggestion
            info!("this system uses the moved arg: {:?}", arg);
```
log rather than `println!` please 😃